packages/network-enablement-controller/tsconfig.json

```
@@ -0,0 +1,22 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "rootDir": "../.."
+  },
+  "references": [
+    {
+      "path": "../../packages/base-controller"
+    },
+    {
+      "path": "../../packages/network-controller"
+    },
+    {
+      "path": "../../packages/multichain-network-controller"
+    },
+    {
+      "path": "../../packages/controller-utils"
+    }
+  ],
```

Nit: What do you think about ?
```suggestion
  "references": [
    { "path": "../base-controller" },
    { "path": "../network-controller" },
    { "path": "../multichain-network-controller" },
    { "path": "../controller-utils" }
  ],